src/main/java/org/frankframework/frankdoc/wrapper/FrankClassDoclet.java

```diff
@@ -307,4 +312,54 @@ public String getJavaDocTag(String tagName) {
 		}
 		return Arrays.asList(tags).stream().map(Tag::text).collect(Collectors.toList());		
 	}
+
+	@Override
+	public FrankMethod[] getDeclaredMethodsAndMultiplyInheritedPlaceholders() {
+		List<FrankMethod> result = new ArrayList<>();
+		result.addAll(frankMethodsByDocletMethod.values());
+		result.addAll(multiplyInheritedMethodPlaceholders);
+		return result.toArray(new FrankMethod[] {});
+	}
+
+	void addMultiplyInheritedMethodPlaceholders() {
+		try {
+			List<FrankClass> interfazes = new TransitiveImplementedInterfaceBrowser<Object>(this).getInterfacesAndTheirAncestors();
```

Comment:
no need not to call this field `interfaces`

Reply:
I renamed variable `interfazes` to `interfaces`.